app/default/content_routes.py

```diff
@@ -33,9 +34,25 @@ def all_questions(fund_short_name, round_short_name):
     )
 
     if fund and round:
+        lang = get_lang()
+
+        # Allow for COF R2 and R3 to use the old mechanism for translating all questions into welsh - the template
+        # for these rounds contains translation tags to build the page on the fly.
+        # All future rounds that need welsh all questions will have them generated from the form json so should
+        # be named with the language in the filename
+        all_questions_prefix = (
+            f"{fund_short_name.lower()}_{round_short_name.lower()[0:2]}"
+        )
+        template_name = f"{all_questions_prefix}_all_questions_{lang}.html"
```

Comment:
Hesitant about the `[0:2]`, for COF which has `R2W2` & `R2W3`  - does this logic apply to both windows, and what about future windows?  Would it be better to use the full round_short_name to clear up this confusion?  Would need to re-name the files though.

Reply:
So this is the existing logic and is there because all windows in the same round have the same set of questions (so far anyway)

Reply:
So if we renamed we'd also have to duplicate files

Reply:
Ah I see, makes sense, guess my worry is we have a new window with different content and it by default uses this - though I think we'd handle that during the development/introduction of the new rounds.